docs/source/api/deliveryservice_server_capabilities.rst

```diff
@@ -0,0 +1,222 @@
+	| sortOrder         | no       | Changes the order of sorting. Either ascending (default or "asc") or descending ("desc")                      |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| limit             | no       | Choose the maximum number of results to return                                                                |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| offset            | no       | The number of results to skip before beginning to return results. Must use in conjunction with limit.         |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| page              | no       | Return the n\ :sup:`th` page of results, where "n" is the value of this parameter, pages are ``limit`` long   |
+	|                   |          | and the first page is 1. If ``offset`` was defined, this query parameter has no effect. ``limit`` must be     |
+	|                   |          | defined to make use of ``page``.                                                                              |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/1.4/deliveryservice_server_capabilities HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+:deliveryServiceID: The :term:`Delivery Service`'s ID
+:xmlID:             The :term:`Delivery Service`'s :ref:`ds-xmlid`
+:lastUpdated:       The date and time at which this association between the :term:`Delivery Service` and the Server Capability was last updated, in an ISO-like format
+:serverCapability:  The Server Capability's name
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; HttpOnly
+	Whole-Content-Sha512: UFO3/jcBFmFZM7CsrsIwTfPc5v8gUiXqJm6BNp1boPb4EQBnWNXZh/DbBwhMAOJoeqDImoDlrLnrVjQGO4AooA==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 07 Oct 2019 22:15:11 GMT
+	Content-Length: 396
+
+	{
+		"response": [
+			{
+				"deliveryServiceID": 1,
+				"lastUpdated": "2019-10-07 22:05:31+00",
+				"serverCapability": "ram",
+				"xmlId": "example_ds-1"
+			},
+			{
+				"deliveryServiceID": 2,
+				"lastUpdated": "2019-10-07 22:05:31+00",
+				"serverCapability": "disk",
+				"xmlId": "example_ds-2
```

Comment:
Seems the commas weren't the only problem. Missing a closing quotation (<kbd>"</kbd>) on this string.

Reply:
These responses weren't copy/pasted from real responses, were they? Or at least not this one. Because if real responses have those errors there's something very strange going on.

Reply:
I didn't copy them from real responses. I got a first draft of these docs from @jheitz200 before the corresponding endpoint code was complete. The JSON syntax errors are my fault. Happy to replace the examples with real output if it is easy to come by once the endpoint code is complete.

Reply:
It's typically pretty easy to copy/paste from something like [the `toaccess` scripts](https://traffic-control-cdn.readthedocs.io/en/latest/tools/toaccess.html) that come bundled with the Python client. There's also a bash script (`infrastructure/cdn-in-a-box/traffic_ops/to-access.sh`) that provides more or less the same behavior - it was originally designed to make the automated setup of CiaB easier, and it was what the `toaccess` scripts were actually based on, but tons of people use the functions therein for manual requests.

Ultimately, though, it doesn't matter where you get the examples, just so long as they're accurate - at least with regard to request URI, request/response bodies, Content-Type and response codes.